lib/publiccloud/provider.pm

```diff
@@ -41,10 +41,34 @@ sub init {
     my ($self) = @_;
     my $file = lc get_var('PUBLIC_CLOUD_PROVIDER');
     assert_script_run('mkdir -p ' . TERRAFORM_DIR);
-    assert_script_run('curl ' . data_url('publiccloud/terraform/' . $file . '.tf') . ' -o ' . TERRAFORM_DIR . '/plan.tf');
+    if (check_var('SLES4SAP_MODE', 'publiccloud')) {
+        my $git_repo = get_var('ha-sap-terraform-deployments');
+        my $cloud_name = $self->get_sap_cloud_name;
+        assert_script_run('cd ' . TERRAFORM_DIR);
+        assert_script_run('git clone --branch ' . get_var('HA_SAP_GIT_TAG', 'master') . ' ' . get_required_var('HA_SAP_GIT_REPO') . ' .');
```

Comment:
```suggestion
        assert_script_run('git clone --branch --depth 1 ' . get_var('HA_SAP_GIT_TAG', 'master') . ' ' . get_required_var('HA_SAP_GIT_REPO') . ' .');
```

Don't fetch history and older commits.  Saves time & space. ;)

Reply:
Good point!

Reply:
Done!